### ISSUE TITLE
prefix should not end with dot

### ISSUE THREAD

**Issue by lilydjwg** [markstory/rust-statsd#6]
[The example from the doc](https://github.com/markstory/rust-statsd/blob/master/src/client.rs#L39-L42) says:

``` rust
use statsd::client::Client;

let client = Client::new("127.0.0.1:8125", "myapp.");
client.incr("some.metric.completed");
```

This will generate a metric named `myapp..some.metric.completed`. The extra dot will cause issues since Graphite treats this the same as `myapp.some.metric.completed` and stores data in the same file, but statsd thinks they are different.

**Comment by markstory** [comment]
I can update the docs.

**Comment by markstory** [comment]
Fixed with 2130ba95b806d9eeea184bfca9ebe421aa7bab84
